### ISSUE TITLE
Create database to store user's high score locally.

### ISSUE THREAD

**Issue by nyoany** [nyoany/games#4]
Include a page that would contain data about the user as well.

**Comment by tttppp** [comment]
What kind of data?! Play style? Cow amicability? Thumb acuracy?  :-D

**Comment by nyoany** [comment]
No, something like if different users are playing on the same device, they would have different accounts with player name, highest score and the date on which they achived that score.

**Comment by tttppp** [comment]
We can use a SharedPreferences object for the storage: https://developer.android.com/reference/android/content/SharedPreferences.html

We can probably do it properly with a db, but if we're only storing a few scores then maybe this is easier?
